src/Symfony/Component/DependencyInjection/composer.json

```diff
@@ -16,7 +16,8 @@
         }
     ],
     "require": {
-        "php": ">=5.5.9"
+        "php": ">=5.5.9",
+        "psr/container": "^1.0@dev"
```

Comment:
add also provide section https://getcomposer.org/doc/04-schema.md#provide , like in https://github.com/Seldaek/monolog/blob/master/composer.json#L52-L54